Extensions/Electron/JsExtension.js

```diff
@@ -0,0 +1,603 @@
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.isFullScreenable');
+
+    extension
+      .addAction(
+        'SetClosable',
+        _('Allow closing'),
+        _('Enables or disables closing of the window by the user.'),
+        _('Enable window closing: _PARAM0_'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .addParameter('yesorno', _('Allow closing?'), '', false)
+      .setDefaultValue('true')
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.setClosable');
+    
+    extension
+      .addCondition(
+        'IsClosable',
+        _('Window closable'),
+        _('Checks if the window can be closed.'),
+        _('The window can be closed'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.isClosable');
+
+    const levelChoices = JSON.stringify([
+      "normal",
+      "floating",
+      "torn-off-menu",
+      "modal-panel",
+      "main-menu",
+      "status",
+      "pop-up-menu",
+      "screen-saver"
+    ]);
+
+    extension
+      .addAction(
+        'SetAlwaysOnTop',
+        _('Make the windows always on top'),
+        _('Puts the window constantly above all other windows.'),
+        _('Make window always on top: _PARAM0_, level: _PARAM1_'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .addParameter('yesorno', _('Enable always on top?'), '', false)
+      .setDefaultValue('true')
+      .addParameter("stringWithSelector", _("Level"), levelChoices, false)
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.setAlwaysOnTop');
+    
+    extension
+      .addCondition(
+        'IsAlwaysOnTop',
+        _('Window always on top'),
+        _('Checks if the window is always on top.'),
+        _('The window is always on top'),
+        _('Advanced window tools/Windows, Linux, MacOS'),
+        'res/actions/window24.png',
+        'res/actions/window.png'
+      )
+      .getCodeExtraInformation()
+      .setIncludeFile('Extensions/Electron/electron-advancedwindowtools.js')
+      .setFunctionName('gdjs.evtTools.electron.isAlwaysOnTop');
+
+    extension
+      .addAction(
+        'SetKiosk',
+        _('Enable kiosk mode'),
+        _('Puts the window in kiosk mode.'),
```

Comment:
Can you explain what Kiosk mode is?

`[...] in kiosk mode. When in kiosk mode, the window will blabla`